### PR TITLE
 updated the french localisation

### PR DESCRIPTION
**Pull request by aur-c**
updated, fixed grammatical, syntax and stylistic errors of the french localisation

### CODE REVIEW

File: _locales/fr/messages.json [hunk 1]
```diff
@@ -422,19 +422,19 @@
     "pin_bookmarks_on_navbar": { "message": "Épingler le lien des marque-pages sur la barre de navigation" },
     "pin_lists_on_navbar": { "message": "Épingler le lien des listes sur la barre de navigation" },
     "me": { "message": "Profil", "description": "You should probably use word 'Profile' in your language" },
-    "tweet_text_font": { "message": "Police d'écriture du Tweet" },
+    "tweet_text_font": { "message": "Police d'écriture du tweet" },
     "use_old_default_profile": { "message": "Utiliser la photo de profil par défaut Oeuf" },
     "colors": { "message": "Couleurs" },
-    "enable_hashflags": { "message": "Activer les icônes de hashtag (peut réduire la vitesse de chargement)" },
+    "enable_hashflags": { "message": "Activer les icônes de mot-clé (peut réduire la vitesse de chargement)" },
```
Comment (charlypoirier) [inline _locales/fr/messages.json:428]: I don't know how Twitter themselves translated it, but we use "hashtag" more than "mot-clé" I think?

Comment (aur-c) [reply _locales/fr/messages.json:428]: i don't like english in french,,

Comment (aur-c) [reply _locales/fr/messages.json:428]: i think mot clé works fine, it's what youtube uses for example

Comment (charlypoirier) [reply _locales/fr/messages.json:428]: yeah alright, I guess it works

**Pull request merged**